docs/premium-features/custom-types-and-actions.md

```diff
@@ -59,70 +61,134 @@ To delete a Module, follow these steps:
 
 ### Create DTO
 
 To create a new DTO, follow these steps:
 
+#### DTO Name and Description
+
 1. Click on the "Add DTO" button in the top right of the APIs tab.
 2. In the dropdown, click on "Add DTO".
-3. Provide a name for your DTO.
-4. Click the "Create DTO" button to finalize the creation process.
+3. Provide a name for your DTO, such as "MyNewDtoName". This will automatically populate the Name field with the provided name.
+4. Optionally, add a description for your DTO.
 
-![Create DTO](./assets/custom-types-and-actions/create-dto.png)
+#### Adding Properties
 
-### Edit, Disable, and Delete DTO
+1. In the "Properties" section, click on "Add property" to define the DTO's properties.
+2. For each property:
+   - Specify the Name of the property (e.g., "firstProperty").
+   - Choose the Type of the property from the dropdown (e.g., "String", "Number", "Boolean", "Date", "JSON", "DTO", "Enum").
+   - If needed, you can change the property to an array by toggling the "Array" switch.
+   - Mark the property as optional by checking the "Optional" checkbox.
+3. Repeat step 2 for each additional property you want to add to the DTO.
+4. If you need to define a union type for a property, click on "+ Add union type" and specify the types for the union.
 
-To edit an existing DTO, click on the DTOs dropdown within its associated Module in the APIs tab. You can modify the DTO's name, description, and members.
+#### Finalizing DTO Creation
 
-To disable an existing DTO, click on the toggle and move it into the disabled state. The dot in the left sidebar will now change colors into Red. You won't be able to use this DTO within the module.
+Click the "Create DTO" button to finalize the creation process.
 
-To delete a DTO entirely, click on the delete button next to the toggle. A confirmation popup will appear. Click on the delete button again to fully delete the DTO.
+After completing these steps, your new DTO will be created within the selected Module, and you can further customize its properties and relationships as needed.
 
-:::note
-Be cautious when deleting DTOs, as it may impact other parts of your application that depend on them.
-:::
+### Create Enum
+
+To create a new Enum, follow these steps:
+
+#### Enum Name and Description
+
+1. Click on the "Add DTO" button in the top right of the APIs tab.
+2. In the dropdown, click on "Add Enum".
+3. Provide a name for your Enum, such as "MyNewEnum". This will automatically populate the Name field with the provided name.
+4. Optionally, add a description for your Enum.
+
+#### Adding Members
+
+1. In the "Members" section, click on "Add member" to define the Enum's members.
+2. For each member:
+   - Specify the Name of the member.
+   - Provide the Value for the member.
+3. Repeat step 2 for each additional member you want to add to the Enum.
+
+#### Finalizing Enum Creation
+
+Click the "Create Enum" button to finalize the creation process.
```

Comment:
Same as previous message- there is no "Create Enum" button to finalize the creation / edition. Is it saved automatically